bin/openresty/build_openresty.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+export RESTY_J="-j8"
+export _RESTY_CONFIG_DEPS=""
+export RESTY_CONFIG_OPTIONS=""
+export RESTY_LUAJIT_OPTIONS=""
+export RESTY_PCRE_OPTIONS=""
```

Comment:
These variables are empty—what's the purpose?

Reply:
It's admirable to consider the future, but all too often the need is non-existent or is misaligned with twhat you'll actually require later. To keep things simple, I suggest removing this part. I'm perfectly okay if you choose to keep it, but I would kindly request that you document it at the very least (the why and how).